### PR TITLE
Remove e2e tests and wrong favicon

### PR DESCRIPTION
**Pull request by ghamaide**

### CODE REVIEW

**Comment by ghamaide** [comment]
@daniel-jorge @savoisn could you approve this PR please ?

File: generators/client/templates/test/e2e/nightwatch.sh [hunk 1]
```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```
Comment (daniel-jorge) [inline generators/client/templates/test/e2e/nightwatch.sh]: This file still used in `generators/client/index.js`. Please update `index.js`.

File: generators/client/templates/test/e2e/createTestUserMongo.txt [hunk 2]
```diff
@@ -1,16 +0,0 @@
-db.users.findAndModify({
```
Comment (daniel-jorge) [inline generators/client/templates/test/e2e/createTestUserMongo.txt]: This file still used in `generators/client/index.js`. Please update `index.js`.

File: generators/client/templates/test/e2e/sample.test.js [hunk 3]
```diff
@@ -1,24 +1,10 @@
-var URL = "https://localhost:8000/<%= applicationFolder %>/";
+const URL = "https://localhost:8000/<%= applicationFolder %>/";
```
Comment (daniel-jorge) [inline generators/client/templates/test/e2e/sample.test.js]: If you delete the e2e tests, this file will no longer be used.

Command `e2e` in `package.json` must also be removed.

**Comment by daniel-jorge** [comment]
👍

**Pull request merged**